openedx/core/djangoapps/site_configuration/migrations/0005_populate_siteconfig_history_site_values.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Written by Troy.
```

Comment:
Nit: Maybe use your Github handle? Or edX email address here? Or just remove this line? Because git knows all...